skynet/modules/ttt/rag/utils.py

```diff
@@ -23,6 +25,34 @@ def split_documents(initial_documents: list[Document]) -> list[Document]:
     return splits
 
 
+def sanitize_filename(filename: str) -> str:
```

Comment:
You should be able to just do os.path.abspath(os.path.realpath(the_path)) and then check if it's inside the folder you want, if not discard it, no?

Reply:
this won't prevent an attacker from accessing files from your system.

Reply:
Not sure I get the vector. If we first get the full path then we can check if that path is allowed no?

Reply:
let's chat on Monday

Reply:
got some time to return to this, you were right